src/integrations/prefect-kubernetes/prefect_kubernetes/utilities.py

```diff
@@ -14,34 +16,35 @@
 V1KubernetesModel = TypeVar("V1KubernetesModel")
 
 
-def enable_socket_keep_alive(client: ApiClient) -> None:
+class KeepAliveClientRequest(ClientRequest):
     """
-    Setting the keep-alive flags on the kubernetes client object.
-    Unfortunately neither the kubernetes library nor the urllib3 library which
-    kubernetes is using internally offer the functionality to enable keep-alive
-    messages. Thus the flags are added to be used on the underlying sockets.
+    aiohttp only directly implements socket keepalive for incoming connections
+    in its RequestHandler. For client connections, we need to set the keepalive
+    ourselves.
+
+    Refer to https://github.com/aio-libs/aiohttp/issues/3904#issuecomment-759205696
     """
 
-    socket_options = [(socket.SOL_SOCKET, socket.SO_KEEPALIVE, 1)]
+    async def send(self, conn: "Connection") -> "ClientResponse":
```

Comment:
if we're actually importing these, we should be able to use them
```suggestion
    async def send(self, conn: Connection) -> ClientResponse:
```

Reply:
true